manifests/overlays/prod/floorist/floorist.enc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: ENC[AES256_GCM,data:TH4=,iv:TpgDZvijVWezbaLg1CHmKthf689wRWDFV3u8hnah0gk=,tag:rGT+SYc2HeZzbVolt9llcQ==,type:str]
```

Comment:
Can you please use the https://github.com/aicoe-aiops/sync-pipelines/blob/master/manifests/.sops.yaml#L2 `encrypted_regex` when sops encrypting files? That way only the creds will be hidden and we can still observe the config in plaintext. It's easier for debugging...

We want to keep all the kubernetes resource metedata in plaintext as well. The idea is that when you look at the config like [this one](https://github.com/aicoe-aiops/sync-pipelines/blob/master/manifests/overlays/prod/usir/secret.enc.yaml) for example you can immediately see the config without the hassle of decrypting it...

Reply:
I've push a new file encrypted with the params provided, please could you check if it looks right this time ?

Thanks ! Also, if it makes sense I'd recommend to add something to the README regarding how to encrypt files with the `sops` CLI, I basically figured it out looking at the `sops` help options looking how to provide GPG keys, I think some mention to it in the README of this project is definitely worth it for newcomers as myself :)